DcCore/DcCore/DC/events.swift

```diff
@@ -78,7 +79,7 @@ public class DcEventHandler {
                         "progress": Int(data1),
                         "error": Int(data1) == 0,
                         "done": Int(data1) == 1000,
-                        "errorMessage": dcContext.lastErrorString as Any,
+                        "errorMessage": self.dcAccounts.get(id: accountId).lastErrorString,
```

Comment:
it is not as getting the context at the start of the function will go away, and then it's actually more efficient, because its only loaded on the events where it is needed.

Reply:
correct solution would be to remove that property completely; it is a relict from times where getLastError() did not exist

/me did not look into other details, currently busy with other things, do not have the correct mindset currently :)

Reply:
good point, yet another thing to cleanup

Reply:
Just looked at it it the value is used in too many places to change in this pr (should stay simple).
Also I'm not sure about changing the location, because `getLastError()` could theoretically  have time races when multiple errors happen, but maybe that's unlikely enough?

Anyways nothing for this pr

Reply:
looking more closely at the code: we were doing a dcAccounts.get() forever, for each event.

so doing this twice in case of DC_EVENT_IMEX_PROGRESS is probably fine - esp. as the other call will probably be removed if we have a global event object.